text/trace/0192-messaging-semantic-conventions-spec.md

```diff
@@ -0,0 +1,165 @@
+# Stabilizing messaging semantic conventions for tracing
+
+This document aims to describe the necessary changes for bringing the [existing semantic conventions for messaging](https://github.com/open-telemetry/opentelemetry-specification/blob/a1a8676a43dce6a4e447f65518aef8e98784306c/specification/trace/semantic_conventions/messaging.md)
+from the current [experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#experimental)
+to a [stable](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#stable)
+state.
+
+This OTEP is based on [OTEP 0173](0173-messaging-semantic-conventions.md),
+which defines basic terms and describes messaging scenarios that should be
+supported by messaging semantic conventions.
+
+## Motivation
+
+This OTEP serves as a foundation for a first stable version of [messaging semantic conventions for tracing](https://github.com/open-telemetry/opentelemetry-specification/blob/a1a8676a43dce6a4e447f65518aef8e98784306c/specification/trace/semantic_conventions/messaging.md).
+It aims to define clear, consistent, and extensible conventions, it also
+describes reasons and motivations that led to the formulation of those
+conventions. The conventions comprise areas such as context propagation, span
+structure, names, and attributes.
```

Comment:
I would also add "trace structure" or "possible relationship between spans" or something similar.
IMO - describing the valid parent-child relationship is a major part of the new spec

Reply:
I intended to cover this by the term "span structure", but I'll replace it with "trace structure" if that's more clear.